### PR TITLE
Updated rules

### PR DESCRIPTION
**Pull request by Nautman**
As per decision in most recent board meeting 2020-07-17.

Please review how I have phrased things and please help to find any spelling errors.

I would also appreciate it if someone could test if the bot will output the new rules and information, perhaps in a dev discord server?

### CODE REVIEW

**Comment by charlesmaddock** [comment]
On line 204, shouldn’t "- Försök att inte vara ett internettroll provocerar fram exempelvis....” be "- Försök att inte vara ett internettroll SOM provocerar fram exempelvis....”?

File: internal/server.go [hunk 1]
```diff
@@ -201,14 +201,22 @@ func (server *Server) InitRules() error {
 	rulesEmbed.AddField("__**Du måste vara medlem i Digital Ungdom**__", "Bli medlem här: https://digitalungdom.se/bli-medlem.\n")
 	rulesEmbed.AddField("__**Använd ditt förnamn**__", "Vi vill gärna skapa en personlig miljö. Använd `/nick`-kommandot. Du kan behålla ditt alias genom ex. `/nick Nikolaus (Jultomten)`.\n")
 	rulesEmbed.AddField("__**Gyllene regeln**__", "Gör mot andra som du själv vill bli behandlad.\n")
-	rulesEmbed.AddField("__**Inget stötande beteende.**__", "- Inga rasistiska, sexistiska, eller aggressiva kommentarer.\n- Inget spam.\n- Undvik gärna svordomar.")
+	rulesEmbed.AddField("__**Inget stötande beteende.**__", "- Skicka inget rasistiskt, sexistiskt, sexualiserande, aggressiva eller i övrigt diskriminerande innehåll.\n- Inget spam.\n- Undvik gärna svordomar.")
+	rulesEmbed.AddField("__**Provocera eller hetsa inte någon.**__", "- Försök att inte vara ett internettroll provocerar fram exempelvis känslomässiga svar, gräl, missförstånd eller utdragna ofruktbara diskussioner som inte leder någon vart.")
```
Comment (Nautman) [inline internal/server.go]:
```suggestion
	rulesEmbed.AddField("__**Provocera eller hetsa inte någon**__", "- Försök att inte vara ett internettroll som provocerar fram exempelvis känslomässiga svar, gräl, missförstånd eller utdragna ofruktbara diskussioner som inte leder någon vart.")
```

**Review by Nautman — COMMENTED** [review]
Yes, you're correct @charlesmaddock! I've added it to my review. Next time, I think you can start a review with GitHub and request changes. That way, I can accept your changes and it will update automatically!

File: internal/server.go [hunk 2]
```diff
@@ -201,14 +201,22 @@ func (server *Server) InitRules() error {
...
 	thinkAbout.SetColor(589568)
 	thinkAbout.AddField("__**Undvik elitism**__", " Var upplyftande gentemot nybörjare! Alla vill lära sig mer och det blir roligare om vi bidrar till varandras utveckling och framsteg. Försök att inte överväldiga nybörjare med en onödigt teknisk jargong.\n")
 	thinkAbout.AddField("__**Använd rätt kanal**__", " Vi har försökt strukturera kanalerna efter ämne, men om du inte känner att något passar kan du antingen skriva till oss eller skriva i #general.\n")
 	thinkAbout.AddField("__**Var gärna tydlig med din nivå**__", "Som nybörjare kan en få mycket hjälp genom att förtydliga att en är ny till vissa områden. Som erfaren kan en känna sig osäker om ens korrespondent egentligen kan mycket mer än en tror!\n")
+	thinkAbout.AddField("__**Sveriges rikes lag gäller**__", "Använd sunt förnuft för att bedöma om ditt beteende är lämpligt. Om ditt beteende bryter mot någon lag så kan du både bli straffad av oss och prövad i en svensk domstol. Ett exempel på olämpligt och olagligt beteende är att hacka någon. Vi vill heller inte att du uppmuntrar sådant beteende!\n")
+
+	punishment := pkg.NewEmbed()
+	punishment.SetTitle("Hur upprätthåller vi reglerna...")
```
Comment (Zigolox) [inline internal/server.go]: Should it be a question or a statement? Right now it is a question without a question mark. Perhaps it should be "Hur vi upprätthåller reglerna..."

**Review by Zigolox — COMMENTED** [review]
One minor fix

File: internal/server.go [hunk 3]
```diff
@@ -201,14 +201,22 @@ func (server *Server) InitRules() error {
...
 	thinkAbout := pkg.NewEmbed()
 	thinkAbout.SetTitle("Tänk på...")
 	thinkAbout.SetColor(589568)
 	thinkAbout.AddField("__**Undvik elitism**__", " Var upplyftande gentemot nybörjare! Alla vill lära sig mer och det blir roligare om vi bidrar till varandras utveckling och framsteg. Försök att inte överväldiga nybörjare med en onödigt teknisk jargong.\n")
 	thinkAbout.AddField("__**Använd rätt kanal**__", " Vi har försökt strukturera kanalerna efter ämne, men om du inte känner att något passar kan du antingen skriva till oss eller skriva i #general.\n")
 	thinkAbout.AddField("__**Var gärna tydlig med din nivå**__", "Som nybörjare kan en få mycket hjälp genom att förtydliga att en är ny till vissa områden. Som erfaren kan en känna sig osäker om ens korrespondent egentligen kan mycket mer än en tror!\n")
+	thinkAbout.AddField("__**Sveriges rikes lag gäller**__", "Använd sunt förnuft för att bedöma om ditt beteende är lämpligt. Om ditt beteende bryter mot någon lag så kan du både bli straffad av oss och prövad i en svensk domstol. Ett exempel på olämpligt och olagligt beteende är att hacka någon. Vi vill heller inte att du uppmuntrar sådant beteende!\n")
+
+	punishment := pkg.NewEmbed()
```
Comment (kelszo) [inline internal/server.go:216]: Where is this being used? Should be sent right?

**Comment by kelszo** [comment]
Good PR, fix sending the punishment embed and the other issue that Simon brought up and I will merge when done.

The PR had the wrong label but it is now fixed.

File: internal/server.go (hunk 3 above)
Comment (Nautman) [reply internal/server.go:216]: Fixed with 4e5a0c71aad61e117d896492b8c442a0cb697352

File: internal/server.go (hunk 2 above)
Comment (Nautman) [reply internal/server.go]:
```suggestion
	punishment.SetTitle("Hur vi upprätthåller reglerna...")
```

**Comment by Nautman** [comment]
Fixed the issues you have mentioned.

**Comment by Nautman** [comment]
I formatted the punishment embed information in a nicer way, as per request from @kelszo! It looks nicer and more inline with our style now. The pictures below were created with the help of an [embed visualizer](https://leovoel.github.io/embed-visualizer/) (by tip from @kelszo!). I think we should be done to merge.

Dark             |  Light
:-------------------------:|:-------------------------:
Screenshot 2020-07-24 at 16 50 29 | Screenshot 2020-07-24 at 16 50 41

**Comment by Zigolox** [comment]
I think this is fine can be merged

**Pull request merged**